labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise01.scala

```diff
@@ -16,14 +16,14 @@ object ListManipulationExercise01 {
    */
   def firstElementInList[T](l: List[T]): T = {
     //buildin
-    null.asInstanceOf[T]
+    return l.head
   }
 
   /**
    * Get the sum of all the elements in the list, e.g. sumOfList(List(1,2,3)) = 6.
    */
   def sumOfList(l: List[Int]): Int = {
-    error("fix me")
+    return l.fold(0)(_ + _)
```

Comment:
Cool! I didn't know you can use the underscore to represent two separate values. There's also a built-in `sum` function.